clusterloader2/testing/measurements-gatherer/config.yaml

```diff
@@ -0,0 +1,141 @@
+#Constants
```

Comment:
It's a nice example how this is used, but I don't think we want to merge it.

Instead, we probably want to migrate existing configs to use it (though not necesary in the same PR).

Reply:
ok sure. will remove it eventually once all other changes are approved. would that be fine?

Reply:
regarding this particular file, i will remove it once everything is reviewed. i am just keeping it handy for my testing.

Reply:
sure - that's fine